index.js

```diff
@@ -0,0 +1,26 @@
+$(document).ready(function() {
+	console.log('hey')
```

Comment:
Happy to see you using `console.log()`s. Just an FYI, when delivering a production site, its recommended you remove any `console.log()`s before final submission.